apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/ExplanationOfBenefitResourceProviderTest.java

```diff
@@ -569,26 +498,26 @@ void testFindByPatientWhenNoClaimsFoundExpectEmptyBundle() {
   }
 
   /**
-   * Verifies that {@link ExplanationOfBenefitResourceProvider#parseTypeParam} ignores an invalid or
-   * unspecified claim type.
+   * Verifies that {@link CommonTransformerUtils#parseTypeParam} ignores an invalid or unspecified
+   * claim type.
    */
   @Test
   void testFindByPatientIgnoresInvalidClaimType() {
     TokenAndListParam listParam = createClaimsTokenAndListParam(Arrays.asList("carriers", "dme"));
-    Set<ClaimType> result = ExplanationOfBenefitResourceProvider.parseTypeParam(listParam);
+    Set<ClaimType> result = CommonTransformerUtils.parseTypeParam(listParam);
```

Comment:
Just noticed this test is actually still testing parseTypeParam. I'll move these few tests along with the others.